index.bs

```diff
@@ -1272,6 +1272,11 @@ the current context. In other words, if the
 allowed to transition from <code>suspended</code> to
 <code>running</code>.
 
+An {{AudioContext}} has an associated <dfn>was allowed to start</dfn> flag,
```

Comment:
Can you explain why we need another flag `was allowed to start`? Can we just look up `is allowed to start` flag on the context when `start()` method is called?

Also it would be better to focus on `start()` and separate out the `resume()` operation into another PR. Or change the title of PR so it can be altogether here.

Reply:
So sorry, this fell off my radar. The reason why I've added `was allowed to start` is that we need to know whether `start()` needs to resume the AudioContext. If there was no previous failed attempt to resume the AudioContext, we should not attempt to resume it.

Reply:
One more question: can we use a document level flag instead of "allowed to start" state in the audio context? (Can't remember the spec proposal)